rust/libnewsboat/src/utils.rs

```diff
@@ -731,6 +752,14 @@ mod tests {
         }
     }
 
+    #[test]
+    fn t_get_auth_method() {
```

Comment:
Coming up with additional tests is fine, but please port the existing ones as well. Having identical tests on both ends serves a few purposes:
* if e.g. Rust suite passes but C++ one fails, we'll know there is a problem with FFI;
* eventually we will remove the FFI wrapper for this function and drop C++ tests for it, which means we need to have a comprehensive test suite in Rust already—otherwise we'll lose test coverage.

Reply:
I forgot to check the C++ tests for this, maybe later on will drop the C++ tests.